views/pages/urls/new.hbs

```diff
@@ -39,6 +39,18 @@
             Shortcode has to be less than 9 characters
           </small>
         </div>
+        {{#ifcontains "admin,employee" user.role}}
```

Comment:
i've updated your role in accounts to employee so you can try out these type of things easily.

Reply:
Until now I just updated the role directly in the db 😜